src/interpret.jl

```diff
@@ -21,7 +21,7 @@ end
 sbmlLog(x) = sbmlLog(10, x)
 sbmlLog(base, x) = log(base, x)
 
-sbmlPower(x, y::Int) = x^float(y)
+sbmlPower(x::Int, y::Int) = x^float(y)
```

Comment:
this still doesn't cover ugly cases like ` 0x0005 ^ Int32(-5)`...will fix :D
```suggestion
sbmlPower(x::Integer, y::Integer) = x^float(y)
```